src/components/db.ts

```diff
@@ -114,10 +114,20 @@ const initUserProfileTable = async (db: Database): Promise<void> => {
             last_updated TIMESTAMP NOT NULL DEFAULT CURRENT_DATE,
             faculty VARCHAR(32),
             program VARCHAR(32),
-            specialization VARCHAR(32)
+            specialization VARCHAR(32),
+            profile_emoji VARCHAR(32)
         )
         `,
   );
+  const columns = await db.all(
+    `SELECT count(*) > 0
```

Comment:
This currently doesn't work; columns always has 1 element, namely `count(*) > 0`. I think you could just do `SELECT * FROM ...`?

Reply:
Okay I'll check this out. Thanks